src/audio/volume.h

```diff
@@ -74,17 +74,17 @@
  * \brief Volume ramp update rate in microseconds.
  * Update volume gain value every 1 ms.
  */
-#define VOL_RAMP_US 1000
+#define VOL_RAMP_UPDATE_US 1000
 
 /**
- * \brief Volume linear ramp length in milliseconds.
- * Use linear ramp length of 250 ms from mute to unity gain. The linear ramp
- * step in Q1.16 to use in vol_work function  is computed from the length.
+ * \brief Macro for volume ramp step computation
+ * Volume ramp step as Q1.16 is computed with equation
+ * step = 2^16 * 1000 / RAMP_UPDATE_US / ramp_length_ms .
```

Comment:
@singalsu is this comment correct? where is ramp_length_ms in the macro below?

Reply:
Yep, need to fix this comment.